### ISSUE TITLE
Add newsletter signup form/button

### ISSUE THREAD

**Issue by torotimes** [working-group-two/wgtwo.com#27]

**Comment by jonny-wg2** [comment]
This needs to be described more...

**Comment by torotimes** [comment]
Yes, I'm working on the concept here (link below). No action needs to be taken on the website yet. Aiming for more clarity next week. 

https://docs.google.com/document/d/1DU_jLtki7aAPss5xkBZyjjDVpxBAX-HJE0NQgPjhkmg/edit

**Comment by torotimes** [comment]
@mtl-wgtwo , can this be easily added as a component on the front page? 

It's a form that adds subscribers to our newsletter distribution database. 

It can be added below the "Adapting to your use case" component. 




<div class style="min-height: calc(100vh - 15rem); padding-bottom: 10rem; padding-top: 5rem; background: #020000;"><div class="l-center-container"><div class="sc-dxgOiQ gDVPix"><h1 size="48px" color="#f5f7f9" class="sc-ckVGcZ jCUICY">Let's stay in touch. Sign up for our newsletter</h1></div><div><div class="sc-bdVaJa iIDDUy"><div><form class="js-cm-form" id="subForm" class="js-cm-form" action="https://www.createsend.com/t/subscribeerror?description=" method="post" data-id="191722FC90141D02184CB1B62AB3DC26955E76E0D27B7A1AB8E959FCE242FF63E7D5C217A62680441C6357AC1AAA7146FDBB1C4EB96CCAB4391278030E63DF48"><div size="base" class="sc-jzJRlG bMslyb"><div size="small" class="sc-jzJRlG liOVdz"><div><label size="0.875rem" color="#5d5d65" class="sc-gzVnrw dEVaGV">Email <span class="sc-dnqmqq iFTUZ">*</span></label><input autoComplete="Email" aria-label="Email" id="fieldEmail" maxLength="200" name="cm-jrduthd-jrduthd" required type="email" class="js-cm-email-input qa-input-email sc-iwsKbI iMsgpL" /></div></div><div size="base" class="sc-jzJRlG bMslyb"><div><p><a size="16px" color="#19a9e5" href="https://www.wgtwo.com/privacy/" rel="noopener" target="_blank" class="sc-hMqMXs drTxYm">Privacy policy</a></p></div></div></div><button size="1rem" color="#fff" type="submit" class="js-cm-submit-button sc-iAyFgw efTFaG">Subscribe</button></form></div></div></div></div></div><style>.iIDDUy { background: rgb(255, 255, 255); border-radius: 0.3125rem; max-width: 35.25rem; margin-left: auto; margin-right: auto; padding: 2.5rem 2.75rem; position: relative; }.ebeRtN { align-items: flex-start; display: flex; }.iHsWXX { font-size: 0.875rem; padding-left: 0.5rem; overflow-wrap: break-word; word-break: break-word; font-family: Helvetica, Arial, sans-serif; color: rgb(67, 77, 93); }.jMsntP { color: rgb(221, 54, 42); font-weight: 400; font-size: 0.75rem; margin-left: 0.125rem; }.eKOoKL { flex-shrink: 0; height: 1rem; position: relative; top: 0.08rem; width: 1rem; }.dEVaGV { font-family: Helvetica, Arial, sans-serif; color: rgb(93, 93, 101); display: block; font-size: 0.875rem; font-weight: 400; margin-bottom: 0.5rem; }.iFTUZ { color: rgb(221, 54, 42); }.iMsgpL { appearance: none; background-color: rgb(255, 255, 255); border: 0px; border-radius: 0.1875rem; box-sizing: border-box; box-shadow: rgba(142, 154, 173, 0.1) 0px 2px 0px 0px inset, rgb(210, 215, 223) 0px 0px 0px 1px inset, rgb(255, 255, 255) 0px 1px 0px 0px; color: rgb(67, 77, 93); font-size: 0.875rem; line-height: 1.5; min-height: 2.8125rem; outline: 0px; padding: 0.75rem 1rem; transition: box-shadow 0.2s ease 0s; width: 100%; }.iMsgpL:focus { box-shadow: transparent 0px 0px 0px 0px inset, rgb(80, 156, 246) 0px 0px 0px 1px inset, rgba(80, 156, 246, 0.25) 0px 0px 0px 2px; }.bMslyb { margin-bottom: 1.5rem; }.liOVdz { margin-bottom: 1rem; }.gDVPix { margin-left: auto; margin-right: auto; max-width: 29.125rem; padding-bottom: 3.125rem; }.iqDtCP { font-size: 3rem; text-align: center; letter-spacing: -1px; line-height: 1.17; -webkit-font-smoothing: antialiased; font-weight: 900; font-family: Lato, sans-serif; color: rgb(0, 0, 0); }.bIkxOz { font-size: 48px; text-align: center; letter-spacing: -1px; line-height: 1.17; -webkit-font-smoothing: antialiased; font-weight: 900; font-family: Lato, sans-serif; color: rgb(244, 246, 248); }.jCUICY { font-size: 48px; text-align: center; letter-spacing: -1px; line-height: 1.17; -webkit-font-smoothing: antialiased; font-weight: 900; font-family: Lato, sans-serif; color: rgb(245, 247, 249); }.fyndJN { display: block; margin: 0px auto; max-width: 100%; max-height: 7.5rem; }.drTxYm { font-size: 16px; font-family: Helvetica, Arial, sans-serif; color: rgb(25, 169, 229); }.bjObOo { border-bottom: 1px solid rgb(210, 215, 223); margin-bottom: 1.5rem; padding-top: 1.5rem; }.czkECw { border-bottom: 1px solid rgb(210, 215, 223); margin-bottom: 0.75rem; padding-top: 0.75rem; }.feTGgs { margin-bottom: 0.25rem; }.efTFaG { background-color: rgb(123, 177, 61); border: none; border-radius: 3px; color: rgb(255, 255, 255); display: inline-block; font-family: Helvetica, Arial, sans-serif; font-size: 1rem; font-style: normal; font-weight: 700; line-height: 1; outline: 0px; padding: 0.75rem 1.5rem; text-decoration: none; transition: background-color 0.1s ease-in 0s, box-shadow 0.1s ease-in 0s; }.efTFaG:hover { cursor: pointer; }.kAkJcy { white-space: pre-wrap; font-size: 16px; font-family: Helvetica, Arial, sans-serif; color: rgb(67, 77, 93); }a, abbr, address, article, aside, b, blockquote, body, caption, cite, code, dd, details, dialog, div, dl, dt, em, fieldset, figcaption, figure, footer, form, h1, h2, h3, h4, h5, h6, header, html, i, iframe, img, label, legend, li, main, menu, nav, object, ol, p, pre, section, small, span, strong, sub, summary, sup, table, tbody, td, tfoot, th, thead, time, tr, u, ul { border: 0px; margin: 0px; padding: 0px; }*, ::after, ::before { box-sizing: inherit; }html { font-family: "Helvetica Neue", Arial, sans-serif; font-size: 16px; }h1, h2, h3, h4, h5, h6, html { color: rgb(67, 77, 93); font-weight: 400; line-height: 1.5; }a { color: rgb(25, 169, 229); text-decoration: underline; }button, label, p { overflow-wrap: break-word; word-break: break-word; }</style><script src="https://ajax.googleapis.com/ajax/libs/webfont/1.6.26/webfont.js"></script><script>WebFont.load({google:{families:['Lato:900:latin','Lato:300:latin','Playfair+Display:700italic:latin','Merriweather:700:latin','Crete+Round::latin','PT+Sans+Narrow:700:latin']}});</script><script type="text/javascript" src="https://js.createsend1.com/javascript/copypastesubscribeformlogic.js"></script>

**Comment by torotimes** [comment]
[newsletter signup form placement.pdf](https://github.com/working-group-two/wgtwo.com/files/8418173/newsletter.signup.form.placement.pdf)
